locale/en/blog/community/building-nodejs-together.md

```diff
@@ -28,17 +28,17 @@ living breathing website whose content is created by our end users and team.
 The website should be the canonical location for documentation on how to use
 Node.js, how Node.js works, and how to find out what's going on in the Node
 community. We have seeded the initial documentation with [how to
-contribute](http://nodejs.org/contribute), [who the core team
-is](http://nodejs.org/about/organization/#index_md_technical_steering_committee),
+contribute](https://nodejs.org/contribute), [who the core team
+is](https://nodejs.org/about/organization/#index_md_technical_steering_committee),
 and some basic documentation of the [project
-itself](http://nodejs.org/about/organization). From there we're looking to
+itself](https://nodejs.org/about/organization). From there we're looking to
 enable the community to come in and build out the rest of the framework for
 documentation.
 
 One of the key changes here is that we're extending the tools that generate API
 documentation to work for the website in general. That means the website is now
 written in markdown. Contributions work with the same
-[pull-request](http://nodejs.org/contribute/code_contributions/) way as
+[pull-request](https://nodejs.org/contribute/code_contributions/) way as
```

Comment:
Should link to https://nodejs.org/en/get-involved/contribute/#code-contributions